### DISCUSSION TITLE
Please help me get AsyncStorage work with Expo

### DISCUSSION THREAD

**Discussion by peter-swe** [react-native-async-storage/async-storage#766] [Q&A] [answered]
I have for weeks tried to make AsyncStorage work, I have googled, I have searched on youtube, and whatever / no matter what I try, I unfortunately do not get it to work ....
1) I have two TextInput
2) the answers stored in players
3) if I console.log (players) I get:
Array [
   Object {
     "id": 1646808104199,
     "number": "7",
     "points": 3,
   },
]
What should I do to be able to store players in AsyncStorage?
Thanks in advance
/ Peter

**Comment by krizzu** [comment]
What exactly problem are you having? We have [documentation for using AsyncStorage](https://react-native-async-storage.github.io/async-storage/docs/usage//), have you had a look there?

**Comment by peter-swe** [comment, marked as answer]
Thanks for the reply!

As https://react-native-async-storage.github.io/async-storage/docs/usage shows, I have tried to store players in AsyncStorage
<img width="749" alt="Skärmavbild 2022-03-16 kl  08 23 42" src="https://user-images.githubusercontent.com/66631035/158540470-84eddaf0-1f0c-4bc5-a429-5b142b443ea2.png">




<img width="690" alt="Skärmavbild 2022-03-16 kl  08 39 40" src="https://user-images.githubusercontent.com/66631035/158542602-c03cca0d-3bde-43c4-8903-d867518bf62b.png">
I get this error when I run saveHandler function:

> JSON.stringify cannot serialize cyclic structures.

As usual, I am grateful for your help and time
with Regards
